src/silo/query_engine/actions/fasta.cpp

```diff
@@ -83,7 +83,7 @@ std::string getTableQuery(
       join_clause += fmt::format(" AND key_table.key = t{}.key", idx);
    }
 
-   const std::string duckdb_table_query = fmt::format(
```

Comment:
So that the move constructor can be chosen. Moves have to mutate the old place holding the object that is moved away. With the const, it copies the object instead; which is what clangd warned me about, hence I fixed it.